workflow/workflow.go

```diff
@@ -418,8 +478,8 @@ const DefaultVersion Version = internal.DefaultVersion
 //	} else {
 //	  err = workflow.ExecuteActivity(ctx, qux, data).Get(ctx, nil)
 //	}
-func GetVersion(ctx Context, changeID string, minSupported, maxSupported Version) Version {
-	return internal.GetVersion(ctx, changeID, minSupported, maxSupported)
+func GetVersion(ctx Context, changeID string, minSupported, maxSupported Version, opts ...internal.GetVersionOptions) Version {
```

Comment:
probably worth mentioning ExecuteWithVersion somewhere in these docs so it's more discoverable, but I have no concrete suggestion at the moment

Reply:
Mentioned it in the GetVersion documentation. Probably, we should merge GetVersion and ExecuteWithVersion doc to one chapter Versioning and point customers there.